indico/modules/events/editing/schemas.py

```diff
@@ -139,6 +140,29 @@ class Meta:
     review_conditions_valid = fields.Boolean()
 
 
+class EditableBasicSchema(mm.ModelSchema):
+    class Meta:
+        model = Editable
+        fields = ('id', 'type', 'state', 'editor')
+
+    state = fields.String()
+    editor = fields.Nested(UserSchema, only=('id', 'avatar_bg_color', 'full_name'))
+
+
+class EditingEditableListSchema(mm.ModelSchema):
+    class Meta:
+        model = Contribution
+        rh_context = ('editable_type')
+        fields = ('id', 'friendly_id', 'title', 'code', 'editable')
+
+    editable = fields.Method('_get_editable')
+
+    def _get_editable(self, contribution):
+        editable_type = self.context['editable_type']
```

Comment:
Why `editable_type` is not available in `self.context`?

Reply:
Look carefully at this line again :)

https://github.com/indico/indico/pull/4460/files#diff-14a5c7d2bef671ab20ad3825f8c7d38aR155

Reply:
No, that is a different error. Fixed that!

Reply:
The `rh_context` not being a tuple would explain why you don't have the `editable_type` in the context though...

Reply:
If I did `rh_context = ('editable_type',)`, still gives error.

Reply:
time to add some print statements directly in `use_rh_args` to debug what's going wrong :)